cli/src/commands/metaedit.rs

```diff
@@ -111,18 +111,18 @@ pub(crate) fn cmd_touch(
     .evaluate_to_commit_ids()?
     .try_collect()?;
     if commit_ids.is_empty() {
-        writeln!(ui.status(), "No revisions to touch.")?;
+        writeln!(ui.status(), "No revisions to modify.")?;
         return Ok(());
     }
     workspace_command.check_rewritable(commit_ids.iter())?;
 
     let mut tx = workspace_command.start_transaction();
     let tx_description = match commit_ids.as_slice() {
         [] => unreachable!(),
-        [commit] => format!("touch commit {}", commit.hex()),
+        [commit] => format!("metaedit commit {}", commit.hex()),
```

Comment:
nit: "edit commit metadata" or something?

`metaedit` might be okay as a command verb, but I think it's cryptic in operation description.

Reply:
Done.